src/commands/test.ts

```diff
@@ -145,10 +145,20 @@ export class TestCommand implements Command {
     },
   ];
 
-  run(_options: CommandOptions, _config: ProjectConfig): Promise<void> {
+  run(_options: CommandOptions, config: ProjectConfig): Promise<void> {
```

Comment:
I need to check if `_options` already been merged into `config`. If not, then there needs to be something below to overwrite settings from `config` with anything in `_options`.